### ISSUE TITLE
Setup codecov

### ISSUE THREAD

**Issue by jamesray1** [Drops-of-Diamond/Diamond-drops#11]
https://github.com/codecov/example-rust

**Comment by samparsky** [comment]
@jamesray1 Seems like this issue has been resolved

**Comment by jamesray1** [comment]
No, it hasn't. I have only added the .travis.yml file, the other steps haven't been done.

**Comment by jamesray1** [comment]
I'm working on this now.

**Comment by jamesray1** [comment]
https://github.com/Drops-of-Diamond/diamond_drops/pull/79

**Comment by jamesray1** [comment]
This looks helpful: https://sunjay.ca/2016/07/25/rust-code-coverage
